common/src/test/java/org/alfresco/hxi_connector/common/repository/filter/CollectionFilterTest.java

```diff
@@ -67,7 +72,22 @@ void testAspectFiltering(boolean expected, Set<String> aspects, List<String> all
         }
     }
 
-    private static Stream<Arguments> provideParameters()
+    @ParameterizedTest
+    @MethodSource("provideAncestorParameters")
+    void testAncestorFiltering(boolean expected, List<String> aspects, List<String> allowed, List<String> denied)
```

Comment:
I assume in args there should be `List<String> ancestors` instead of `List<String> aspects`

Reply:
Thanks, fixed that.